### PR TITLE
Add eslint rule and rename

### PR DESCRIPTION
**Pull request by sonnyhu**
Added `eslintrc-frontend-shared.yml`, `eslintrc-shared.yml` and updated `README.md`

### CODE REVIEW

**Comment by dlsmith** [comment]
Maybe throw in some top-level directories, `eslint` and `jscs`? `rules`, for example, is jscs-specific.

File: eslint/eslintrc-frontend-shared.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+extends:
+    "./eslintrc-shared.yml"
+
+rules:
+    # 2 means error, 1 means warning, 0 means ignore.
+
+    # JSX.
+    jsx-quotes: 2
+
```
Comment (paulmelnikow) [inline eslint/eslintrc-frontend-shared.yml]: Now that this is separated out, can you mention `eslint-plugin-react` is required, and include a link?

**Comment by paulmelnikow** [comment]
One documentation note, otherwise :+1:

Thanks for taking care of this!

**Comment by paulmelnikow** [comment]
Can you also update the package name in `package.json` to match the repo?

**Comment by sonnyhu** [comment]
Merged via fd87517

**Pull request closed without merging**